NOTICE.txt

```diff
@@ -3976,77 +4047,6 @@ If the Work includes a "NOTICE" text file as part of its distribution, then any
 
 END OF TERMS AND CONDITIONS
 
-
----
-
-## webpack
```

Comment:
(question) why is webpack removed? is it not a requirement to include their licence anymore?

Reply:
I didn't see it in the package.json, but I've added it back now.